server/src/test/java/io/crate/integrationtests/PartitionedTableIntegrationTest.java

```diff
@@ -2364,4 +2364,38 @@ public void test_select_partitioned_by_column_with_query_then_fetch_plan() throw
             "1| Trillian\n"
         ));
     }
+
+    @Test
+    @UseRandomizedSchema(random = false)
+    public void test_alter_table_drop_constraint_on_partitioned_table() throws Exception {
```

Comment:
this is failing on master, so I think makes sense to have dedicated test.

Reply:
Can we add a comment that this validates a previous bug?